packages/create/src/generators/linting-eslint-ts/templates/_.eslintrc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     'plugin:import/errors',
     'plugin:import/warnings',
+    'plugin:import/typescript',
```

Comment:
does adding this create any additional linting errors?

Reply:
Not entirely sure but I think it was what triggered the `Missing return type on function.eslint@typescript-eslint/explicit-module-boundary-types` linting error on the `render()` method.